src/Scene/AnimationPlayer.js

```diff
@@ -115,6 +123,9 @@ class AnimationPlayer {
      */
     play(animation) {
         this.animation = animation;
+        this.dispatchEvent({
+            type: 'playAnimation',
```

Comment:
`animation-started` ? (events are things that happened, so they can really start with a verb)